eventuals/lock.h

```diff
@@ -1,6 +1,7 @@
 #pragma once
 
 #include <atomic>
+#include <list>
```

Comment:
This looks unnecessary.

Reply:
Oops, remnants of an earlier implementation, THANK YOU!